lib/attr.gd

```diff
@@ -37,8 +37,14 @@
 ##
 ##  <Description>
 ##  This info class (together with <Ref InfoClass="InfoWarning"/> is used
-##  for messages about attribute storing  being  disabled  (at  level  2)  or
-##  enabled (level 3). It may be  used  in  the  future  for  other  messages
+##  for messages about attributes. The current warnings are:
+##  <List>
+##    <Item> Attribute storing  being  disabled  (level  2) </Item>
+##    <Item> Atribute storing being enabled (level 3)</Item>
+##    <Item> Trying to assign a non-mutable attribute a different value (level 3)</Item>
+##    <Item> An attribute which is marked as assigned, but has no value (level 3)</Item>
+##  </List>
+##  It may be  used  in  the  future  for  other  messages
 ##  concerning changes to attribute behaviour.
```

Comment:
That last sentence seems redundant to me. Of course we may add new Info messages in the future, that's true for every info class, isn't it? 

So, optional change: perhaps remove it?

Reply:
So, this, along with quite a few of your other comments, are text which already existed here. I'm happy to change it of course.

Reply:
Yeah, that's why I said this optional -- I am not expecting you to fix text that was already broken, although of course it is much appreciated if you do :-)